src/components/Overview/PercentageOverview.js

```diff
@@ -0,0 +1,66 @@
+import { CircularProgressDo } from '../UI/PercentageOverview/CircularProgressDo.styled';
+import { Container } from '../UI/PercentageOverview/Container';
+import { CircleTodo } from '../UI/PercentageOverview/CircleTodo.styled';
+import { CircleTodoBg } from '../UI/PercentageOverview/CircleTodoBg.styled';
+import { CircleText } from '../UI/PercentageOverview/CircleText.Styled';
+import { useEffect, useState } from 'react';
+import { InfoBox } from '../UI/PercentageOverview/InfoBox.styled';
+import { Headline } from '../UI/PercentageOverview/Headline.styled';
+import { InfoText } from '../UI/PercentageOverview/InfoText.styled';
+
+export default function PercentageOverview({
+	boxShadow,
+	size,
+	strokeWidth,
+	percentage,
+	color,
+	bgColor,
+	headline,
+	proportion,
+	sum,
+}) {
+	const [progress, setProgress] = useState(0);
+	useEffect(() => {
+		setProgress(percentage);
+	}, [percentage]);
+
+	const viewBox = `0 0 ${size} ${size}`;
+	const radius = (size - strokeWidth) / 2;
+	const circumference = radius * Math.PI * 2;
+	const dash = (progress * circumference) / 100;
```

Comment:
Maybe I missed sth but I don't think you need the `useState` and `useEffect`: 

```suggestion
	const viewBox = `0 0 ${size} ${size}`;
	const radius = (size - strokeWidth) / 2;
	const circumference = radius * Math.PI * 2;
	const dash = (percentage * circumference) / 100;
```